src/getting-started/logging-in.rst

```diff
@@ -80,4 +80,3 @@ Next steps
 ==========
 
 -  Learn more about the `JupyterLab interface <https://jupyterlab.readthedocs.io/en/latest/>`_ (external Project Jupyter documentation).
--  :doc:`Explore demo notebooks <notebook-demo>` featuring LSST data analysis.
```

Comment:
In the future it'd be good to link to docs for the new notebooks, but this is fine for the immediate fix.